archlinuxcn/v2ray-desktop/lilac.yaml

```diff
@@ -0,0 +1,9 @@
+maintainers:
+  - github: Dr-Incognito
```

Comment:
You don't have a public email on your github profile page, so you need to specify your email address here.